plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftConfig.java

```diff
@@ -69,4 +71,30 @@ public RedshiftConfig setUnloadIamRole(String unloadIamRole)
         this.unloadIamRole = unloadIamRole;
         return this;
     }
+
+    public Optional<@Pattern(regexp = "^s3://[^/]+(/[^/]+)?$", message = "Path shouldn't end with trailing slash") String> getBatchedInsertsCopyLocation()
+    {
+        return Optional.ofNullable(batchedInsertsCopyLocation);
+    }
+
+    @Config("redshift.batched-inserts-copy-location")
+    @ConfigDescription("A writeable location in Amazon S3, to be used to stage data for using Redshift COPY FROM statement during batched inserts")
+    public RedshiftConfig setBatchedInsertsCopyLocation(String batchedInsertsCopyLocation)
+    {
+        this.batchedInsertsCopyLocation = batchedInsertsCopyLocation;
+        return this;
+    }
+
+    public Optional<String> getBatchedInsertsCopyIamRole()
+    {
+        return Optional.ofNullable(batchedInsertsCopyIamRole);
+    }
+
+    @Config("redshift.batched-inserts-copy-iam-role")
```

Comment:
Do we need a toggle to control that we enable the COPY FROM or not, otherwise we only control it on session level?